pkg/config/remote/uptane/client.go

```diff
@@ -66,6 +68,7 @@ func NewClient(cacheDB *bbolt.DB, cacheKey string, orgID int64) (*Client, error)
 func (c *Client) Update(response *pbgo.LatestConfigsResponse) error {
 	c.Lock()
 	defer c.Unlock()
+	c.cachedVerify = false
```

Comment:
For the first attempt this looks 👍🏻 .

Right now it looks like we go through Uptane verification even if nothing has changed in the TUF state since there's nothing in the TUF client Update functions that let us know nothing has changed. (And we're not inspecting the response to detect this ourselves either.) If verification is expensive, should we try to do better here and detect that nothing has changed in the TUF state to prevent going through verification whatsoever when there's no need to?

Reply:
We'll need to Dario's change to address this properly as it's currently very hard to understand if an update modifies the state or not with how opaque the TUF client is.

Reply:
Shouldn't we also invalidate the cache at the beginning of `func (c *Client) TargetsMeta()`?